env.js

```diff
@@ -113,6 +113,7 @@ function setStorage() {
   env.profile_collection = readENV('MONGO_PROFILE_COLLECTION', 'profile');
   env.devicestatus_collection = readENV('MONGO_DEVICESTATUS_COLLECTION', 'devicestatus');
   env.food_collection = readENV('MONGO_FOOD_COLLECTION', 'food');
+  env.azurepush_collection = readENV('MONGO_AZUREPUSH_COLLECTION', 'azurepush');
```

Comment:
OK, agree it will be harder to change later so makes sense to try making it more generic now